packages/bootstrap-4/src/ErrorList/ErrorList.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+
+import Card from "react-bootstrap/Card";
+import ListGroup from "react-bootstrap/ListGroup";
+
+import { ErrorListProps } from "@rjsf/core";
+
+const ErrorList = ({ errors }: ErrorListProps) => (
+  <Card border="danger" className="mb-2">
+    <Card.Header className="alert-danger">Errors</Card.Header>
+    <Card.Body className="p-0">
+      <ListGroup >
+        {errors.map((error, i: number) => {
+          return (
+            <ListGroup.Item key={i} className="border-0">
+              <small>{error.stack}</small>
```

Comment:
Let's use regular-sized text for ErrorList

```suggestion
              <span>{error.stack}</span>
```